README.md

```diff
@@ -128,8 +128,10 @@ Users should prefer the statically inspectable metadata above, but if you need
 the full power and flexibility of arbitrary runtime predicates... here it is.
 
 We provide a few predefined predicates for common string constraints:
-`IsLower = Predicate(str.islower)`, `IsUpper = Predicate(str.isupper)`, and
-`IsDigit = Predicate(str.isdigit)`.
+- `IsLower = Predicate(str.islower)`
+- `IsUpper = Predicate(str.isupper)`
+- `IsDigit = Predicate(str.isdigit)`
+- `IsFinite = Predicate(math.isfinite)`
```

Comment:
We document this as if we're just defining the annotation, but actually define the annotated type.

Reply:
Agreed, I kept the existing format, but I'm not sure if that was an affirmation, question, or suggestion?

Reply:
More of a thoughtless observation, really 😓 

Coming back with some more thought, I'm concerned that we're misleading people a bit with the docs - as written, I would expect to write `Annotated[float, IsFinite]` rather than `IsFinite[float]` - but my understanding is that the latter is actually required given our implementation so far, right?  If so, I think we should change the docs.

Reply:
Yes, or we can just change the `=` to `:` and put an example there:
```
- `IsFinite`: `Predicate(math.isfinite)`

These predicates are used as `x: IsFinite[float]` which is equivalent to `x: Annotated[float, Predicate(math.isfinite)]` (that is, `IsFinite` is just a generic type alias for your convenience).
```